### PR TITLE
Calibration

### PR DESCRIPTION
**Pull request by dNechita**
Add calibration feature for AD96TOF1-EBZ board.

Addresses #187

### CODE REVIEW

File: calibration/Readme.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+The python based calibration software is supported on the DragonBoard 410c Broad Market Kit as well as Windows 10. It is highly recommended to run the calibration software on the Dragonboard platform.
+
+# Dragonboard Install
+The basic setup of the Broad Market Kit is described [here](https://github.com/analogdevicesinc/aditof_sdk).
+
+Clone this SDCARD image to a 16GB SDCARD and put the SD Card slot of the Dragonboard and power on the board.
+
+## Run Sweep Calibtation For A Given Mode
```
Comment (grigorovici-andreea) [inline calibration/Readme.md]: Run sweep **calibration**

File: calibration/Readme.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Modify cal_replace.json for appropriate firmware paths.
+
+```
+cd scripts
+sudo ../../py36tofcalib/bin/python eeprom_replace_cal.py cal_replace.json
+
+```
+
+## Controlling logger output
+The logger.json allows you configure the logger output.You can set the appropriate logging level. Here's good discription: [logging levels](https://docs.python.org/3.6/library/logging.html#levels)
```
Comment (grigorovici-andreea) [inline calibration/Readme.md]: **description** not discription

File: calibration/Readme.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+https://github.com/adi-sdg/tof_sdk/tree/master/IO_Library/python
+
+
+## Setup Environment for Sweep Calibration
+
+* Plug ADI TOF camera module to a host (Windows, Linux or OSX)
+* Make sure the ADI camera is recognized by the host
+* Set a target - a plain white board within 20cm to 200 cm in front of the camera - the distance between the target and the edge of camera lens is the target_distance 
+
+## Run Sweep Calibtation For A Given Rang or Mode
```
Comment (grigorovici-andreea) [inline calibration/Readme.md]: again, **calibration**,  or do you refer to something else?

File: calibration/tof_calib/write_to_lf_file.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+    linear_offset_df['reg_offset_value_hex'] = linear_offset_df['corrected_offset'].apply(lambda x: tohex(x, 14, '04x'))
+
+    replace_value_lf_files(linear_offset_df['reg_offset_value_hex'] , r'.*LNR_OFST.*',file_path, file_path1)
+    
+    #print(linear_offset_df['gain'][0])
+    gain_h = to_fixed_point(linear_offset_df['gain'][0])
+    offset_h = tohex(int(linear_offset_df['offset'][0]), 15, '04x')
+    
+    #print(gain_h)
```
Comment (grigorovici-andreea) [inline calibration/tof_calib/write_to_lf_file.py]: do we need these commented print lines?

File: calibration/tof_calib/write_to_lf_file.py [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+def replace_value_lf_files(reg_values, pattern, input_file, output_file):
+    ind = 0
+    with open(input_file) as f_in:
+        with open(output_file, 'w') as f_out:
+            pattern = re.compile(pattern)
+            for line in f_in:        
+                m = re.search(pattern, line)
+                if m: 
+                   addr, value, marker, comment = m.group(0).split()
+                   #print('addr: ' + str(addr) + ' value:' + str(value))
```
Comment (grigorovici-andreea) [inline calibration/tof_calib/write_to_lf_file.py]: do we need these commented print lines?

File: calibration/tof_calib/write_to_lf_file.py [hunk 6]
```diff
@@ -0,0 +1,126 @@
+from tempfile import mkstemp
+import os
+import re
+import numpy as np
+import logging
+
+# def float_bin(number, places = 3):
```
Comment (grigorovici-andreea) [inline calibration/tof_calib/write_to_lf_file.py]: why do we keep this commented function?

File: calibration/Readme.md (hunk 1 above)
Comment (dNechita) [reply calibration/Readme.md]: This is outdated. Take a look at the other commits.

**Pull request merged**